pkg/config/piped.go

```diff
@@ -928,7 +928,8 @@ func (n *NotificationReceiver) Mask() {
 
 type NotificationReceiverSlack struct {
 	HookURL           string   `json:"hookURL"`
-	OAuthToken        string   `json:"oauthToken"`
+	OAuthTokenData    string   `json:"oauthTokenData"`
```

Comment:
```suggestion
        OauthToken        string   `json:"oauthToken"`
	OAuthTokenData    string   `json:"oauthTokenData"`
```

Please use [Deprecated](https://github.com/golang/go/wiki/Deprecated)

Reply:
Done